src/commands/dev.ts

```diff
@@ -216,16 +216,19 @@ export default class Dev extends BaseCommand {
   }
 
   private async runLocal() {
-    const { args, flags } = await this.parse(Dev);
+    const { args, flags, raw } = await this.parse(Dev);
```

Comment:
I do not think we need to use the raw data here. Flags can be set with the multiple option to true.
https://oclif.io/docs/flags

```
    'secrets-file': Flags.string({
      description: 'Path to a secrets file',
      multiple: true,
      default: [],
    }),
```

Then if you went to access the flag you can just do. `flags['secrets-file']` to get a string array. This will also make sure the flag will appear in our readme.